crates/aptos/src/common/types.rs

```diff
@@ -1318,6 +1334,26 @@ impl TransactionOptions {
             gas_unit_price
         };
 
+        // Get sequence number for account
+        let (account, state) = get_account_with_state(&client, sender_address).await?;
+        let sequence_number = account.sequence_number;
+
+        // Retrieve local time, and ensure it's within an expected skew of the blockchain
+        let now = SystemTime::now()
+            .duration_since(UNIX_EPOCH)
+            .map_err(|err| CliError::UnexpectedError(err.to_string()))?
+            .as_secs();
```

Comment:
don't we already have something in infallible?

Reply:
We... probably do

Reply:
The only thing is the infallible then doesn't give a user friendly error (that's in the same format as everything else), so I'm leaving this